src/01-functions.js

```diff
@@ -112,7 +129,13 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b) {
+  if(typeof a === typeof b){
+    return true
+  } else {
+return false
```

Comment:
Let's watch our spacing here.  Also recall that if we do not return true in line 134 - we don't neeed an else as the only other option is a return of false